examples/resources/netlify_waf_policy/import.sh

```diff
@@ -0,0 +1,2 @@
+# Import a WAF policy by its team ID and the policy ID
+terraform import netlify_log_drain.http 6600abcdef1234567890abcd:6600abcdef1234567890abcd
```

Comment:
Not sure what this is doing?

Reply:
It should say netlify_waf_policy 🙂 It's an example of how you import it through the CLI. It goes into the generated Markdown docs.